packages/x-components/static-docs/experience-search-and-discovery/empathize.md

```diff
@@ -11,7 +11,9 @@ experience that is usually displayed below the search box. Consider it the natur
 
 ![Empathize](/assets/media/xcomponents_func_empathize.gif)
 
-**Placement matters** One of the characteristics of X&nbsp;Components is that you can place them
+**Placement matters** 
+
```

Comment:
Can we get rid of this blank line?

```
**Placement matters**
One of the characteristics...
```
Otherwise, we can fix it by removing the sub-title **Placement matters**

Reply:
We need the extra line break to avoid lint to collapse lines 14 and 16 in the14...